ddsc/ddsclient.py

```diff
@@ -297,6 +297,30 @@ def run(self, args):
             self.remote_store.delete_project_by_name(args.project_name)
 
 
+class ListAuthRolesCommand(object):
+    """
+    List available auth roles for a project.
+    """
+    def __init__(self, config):
+        """
+        Pass in the config who can create a remote_store so we can access the remote data.
+        :param config: Config global configuration for use with this command.
+        """
+        self.remote_store = RemoteStore(config)
+
+    def run(self, args):
+        """
+        Prints out non deprecated auth roles.
+        :param args Namespace arguments parsed from the command line
+        """
+        auth_roles = self.remote_store.get_active_auth_roles(RemoteAuthRole.PROJECT_CONTEXT)
```

Comment:
Is it worthwhile to document why we're only listing auth roles for projects? I saw https://github.com/Duke-Translational-Bioinformatics/duke-data-service/issues/665, and it sounds like there was some confusion, so it might make sense to convey why the ListAuthRolesCommand only returns project auth roles.